homeassistant/components/sensibo/switch.py

```diff
@@ -166,3 +178,15 @@ async def async_turn_on_off_pure_boost(self, key: str, value: Any) -> bool:
             data["primeIntegration"] = False
         result = await self._client.async_set_pureboost(self._device_id, data)
         return bool(result.get("status") == "success")
+
+    @async_handle_api_call
+    async def async_turn_on_off_smart(self, key: str, value: Any) -> bool:
+        """Make service call to api for setting Climate React."""
+        if self.device_data.smart_on is None:
+            raise HomeAssistantError(
```

Comment:
This doesn't seem to hit on this code path.

![sensibo_testing](https://user-images.githubusercontent.com/663432/197411880-8176f359-259a-4300-9d5b-cdb0f2a0e82f.gif)

Reply:
Unrelated, but I think Temperature feels like 22.2 °F might be an F/C conversion issue

Reply:
> This doesn't seem to hit on this code path.

Not entirely sure what you mean. You don't have climate react enabled before so you would expect to get the error?

Reply:
> Not entirely sure what you mean. You don't have climate react enabled before so you would expect to get the error?

Right. I'd expect it to raise an exception here instead of flipping back off with no feedback

Reply:
> Unrelated, but I think Temperature feels like 22.2 °F might be an F/C conversion issue

#80843

Reply:
Try again with the last changes. Can't test myself as once climate react has been configured it can never be empty again.